kube/hof-rds-api/ingress.yml

```diff
@@ -23,13 +23,13 @@ spec:
         - data-service-{{ .DRONE_SOURCE_BRANCH }}.branch.sas-notprod.homeoffice.gov.uk
       {{ end }}
       {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
-      secretName: branch-tls-external
-      {{ else }}
+      secretName: branch-tls-internal
+      {{ else }} 
```

Comment:
Space added on end of line here...probably ok but hopefully it won't mess with the yaml